code-assurance-playbook.Rproj

```diff
@@ -1,4 +1,5 @@
 Version: 1.0
+ProjectId: 428199eb-0aaa-43fb-b0e4-791fbb1ffb55
```

Comment:
See [this thread](https://forum.posit.co/t/use-for-projectid-in-rproj-file/196345) on what this is and why it is in commit